deltachat-ios/AppDelegate.swift

```diff
@@ -55,7 +55,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         let console = ConsoleDestination()
         console.format = "$DHH:mm:ss.SSS$d $C$L$c $M" // see https://docs.swiftybeaver.com/article/20-custom-format
         logger.addDestination(console)
-        dcContext.logger = DcLogger()
+
+        dcAccounts.openDatabase()
+        migrateToDcAccounts()
+        if dcAccounts.getAll().isEmpty {
+            dcAccounts.add()
```

Comment:
results in linter warning `Result of call to 'add()' is unused`. however, if that fails we cannot do much but thowing a fatal error to the user as needed at https://github.com/deltachat/deltachat-ios/pull/1272/files?diff=unified&w=1#diff-fc4377fbb4c60bfc9a11f53fd448de91bbf34787141509ddb85242515fea9e23R469 - do we have a function to show such an fatal error?

Reply:
No, we don't have such a thing. Normally we're throwing fatal errors (like exceptions in Java world). So the app would just close and we would see the crashes in the reports.

Reply:
we could do sth. like
```
if dcAccounts.getAll().isEmpty, dcAccounts.add() == 0 {
      fatalError("Could not initialize a new account.")
}
```